compiler/passes/convert-uast.cpp

```diff
@@ -1163,6 +1215,15 @@ struct Converter {
       } else if (name == borrowedStr) {
         ret = new CallExpr(PRIM_TO_BORROWED_CLASS_CHECKED);
       }
+
+    // Unwrap 'foo.bytes' specially to avoid wrapping it in a third call.
```

Comment:
Is this specific to `bytes` somehow? Are all other dot expressions handled so they don't create a third call?

Reply:
This code is just handling the situation that we are calling a keyword, which isn't represented in the normal way, in the production AST anyway. Other  regular dot expressions are handled elsewhere in this file (David could tell you where).

Reply:
TIL `bytes` gets its own token

Reply:
Yeah, we have many of the builtin types have their own token so we treat them as "keywords". Mainly we don't want people to do things like `var bytes = 23` but it is of course debateable whether handling that in the parser was the right move.